src/main/kotlin/app/revanced/patches/twitter/misc/fxtwitter/FxTwitterPatch.kt

```diff
@@ -0,0 +1,134 @@
+package app.revanced.patches.twitter.misc.fxtwitter
+
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.getInstructions
+import app.revanced.patcher.extensions.InstructionExtensions.replaceInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patcher.patch.options.types.StringPatchOption.Companion.stringPatchOption
+import app.revanced.patches.twitter.misc.fxtwitter.fingerprints.FxTwitterPatchFingerprint
+import app.revanced.patches.twitter.misc.fxtwitter.fingerprints.FxTwitterPatchFingerprintResource
+import com.android.tools.smali.dexlib2.Opcode
+import com.android.tools.smali.dexlib2.builder.BuilderInstruction
+import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
+import com.android.tools.smali.dexlib2.iface.instruction.ReferenceInstruction
+import com.android.tools.smali.dexlib2.iface.instruction.formats.Instruction35c
+import com.android.tools.smali.dexlib2.iface.reference.MethodReference
+import com.android.tools.smali.dexlib2.iface.reference.StringReference
+import java.util.logging.Logger
+
+
+@Patch(
+    name = "FxTwitter",
+    description = "Replaces the default Twitter share link with FxTwitter.",
+    compatiblePackages = [CompatiblePackage("com.twitter.android")]
+)
+@Suppress("unused")
+object FxTwitterPatch : BytecodePatch(
```

Comment:
The patch is not exclusive to FxTwitter. It should be called something along "Change share link domain"